bitnami/prestashop/values.yaml

```diff
@@ -477,6 +547,50 @@ metrics:
     prometheus.io/scrape: "true"
     prometheus.io/port: "9117"
 
+# Add custom certificates and certificate authorities to redmine container
+certificates:
+  customCertificate:
+    certificateSecret: ""
+    chainSecret: {}
+    # name: secret-name
+    # key: secret-key
+    certificateLocation: /etc/ssl/certs/ssl-cert-snakeoil.pem
+    keyLocation: /etc/ssl/private/ssl-cert-snakeoil.key
+    chainLocation: /etc/ssl/certs/mychain.pem
```

Comment:
Could you these paths?

Reply:
This is using the defaults, basing on the work in #2774

> Adds a snakeoil certificate to the container. Used can be used for TLS sessions to the ingress.